.github/workflows/test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           { fullName: 'material.blue.dark.compact', name: 'material.dark' }
         ]
         ARGS: [
-          { project: 'angular', page: 'crm-contact-form' },
```

Comment:
These are the names of the test files from the testing package. Rename the 'crm-contact-form.test.js' file as well.